### DISCUSSION TITLE
Hello! Please let me know your favorite Kanji character!

### DISCUSSION THREAD

**Discussion by vincentmichael089** [NAIST-SE/AttractingContributors#486] [Show and tell]
Hello! I want to gather everyone's favourite kanji! 
Please feel free to update the list!! 😊😊
よろしくお願いします！

https://github.com/vincentmichael089/AttractingContributors-Kanji

**Comment by ploychanok** [comment]
I’ve added a kanji ‘niku’ and made a pull request.
Would you mind contribute mine too https://github.com/ploychanok/AttractingContributors-WithinThreeWords

**Reply by vincentmichael089** [reply]
Sure!!

**Comment by Hidenori333** [comment]
I have added 忘 and made a pull request. 
I appreciate if you continue for mine as well https://github.com/Hidenori333/AttractingContributors-favorite-movies

**Reply by vincentmichael089** [reply]
I contributed to your repository! Thank you very much!
